src/app/plugins/platform/JavaScript/templates/_eslintrc.js

```diff
@@ -50,6 +53,11 @@ module.exports = {
             "error",
             "as-needed"
         ],
+
+        // TS already checks this and it causes problems
+        // https://github.com/yannickcr/eslint-plugin-react/issues/2353#issuecomment-513036364
+        "react/prop-types": 0,
```

Comment:
Without this you have to use the props interface redundantly: https://github.com/yannickcr/eslint-plugin-react/issues/2353#issuecomment-534752036 or rewrite as function declarations.